include/boost/math/distributions/arcsine.hpp

```diff
@@ -85,7 +86,11 @@ namespace boost
       { // Check x_min < x_max
         if (x_min >= x_max)
         {
+          #ifndef BOOST_MATH_STANDALONE
           std::string msg = "x_max argument is %1%, but must be > x_min = " + lexical_cast<std::string>(x_min) + "!";
```

Comment:
@jzmaddock : Does this still need lexical_cast? It looks like we could just get away with `std::to_string`. . .

Reply:
+1, I think that's correct, yes.  And it allows us to remove the #if logic entirely.

Reply:
`std::to_string` will only work with the builtin arithmetic types. I can keep the else block which will pass tests using `real_concept` but offer a less helpful error message.

Reply:
Good point, glad one of us is paying attention! :)